Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridViewButtonCell.cs

```diff
@@ -250,15 +259,24 @@ public ButtonStyle ButtonStyle
                             UseColumnTextForButtonValue: true
                         } col) && !kDgv.Rows[rowIndex].IsNewRow)
                     {
-                        _shortTextValue.ShortText = col.Text;
+                        if (_shortTextValue != null)
+                        {
+                            _shortTextValue.ShortText = col.Text;
+                        }
                     }
-                    else if (!string.IsNullOrEmpty(FormattedValue?.ToString()))
+                    else if (FormattedValue != null ? !string.IsNullOrEmpty(FormattedValue.ToString()) : false)
```

Comment:
Not sure this change adds anything, a null check is done in the string checker ??

Reply:
Reverted